### PR TITLE
chore: new bug template

### PR DESCRIPTION
**Pull request by Marcellofabrizio**
This PR adds the GitHub bug Issue template as requested in #587.

Let me know of any sections that must be added.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: Shell
```
Comment (arthurfiorette) [inline .github/ISSUE_TEMPLATE/bug_report.yml:77]: trailling :)

```suggestion
      render: Shell

```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      required: true
+  - type: input
+    id: storage
+    attributes:
+      label: What storage is being used
+      description: What storage is being used?
+      placeholder: Web Storage
+    validations:
+      required: true
+  - type: dropdown
```
Comment (arthurfiorette) [inline .github/ISSUE_TEMPLATE/bug_report.yml:62]: We can merge this with the NodeJS

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    attributes:
+      label: Axios Cache Interceptor Version
+      description: What version of our software are you running?
+      placeholder: 0.9.3
+    validations:
+      required: true
+  - type: input
+    id: node_version
+    attributes:
+      label: Node Version
```
Comment (arthurfiorette) [inline .github/ISSUE_TEMPLATE/bug_report.yml:41]:
```suggestion
      label: Node / Browser version
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      description: What version of Node.js are you running?
```
Comment (arthurfiorette) [inline .github/ISSUE_TEMPLATE/bug_report.yml:42]:
```suggestion
      description: What version of Node.js/Browser are you running?
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+      placeholder: 16.9.1
```
Comment (arthurfiorette) [inline .github/ISSUE_TEMPLATE/bug_report.yml:43]:
```suggestion
      placeholder: Node v16.9.1 or Chrome 98
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 6]
```diff
@@ -0,0 +1,77 @@
+name: Bug report
+description: Create a report to help us improve
+title: 'bug: something is wrong :('
```
Comment (arthurfiorette) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]:
```suggestion
title: 'Bug: '
labels: ["bug", "triage"]
```

Created a triage label

**Review by arthurfiorette — CHANGES_REQUESTED** [review]

**Comment by arthurfiorette** [comment]
Suuuuper thanks for your work!!

**Comment by Marcellofabrizio** [comment]
> Suuuuper thanks for your work!!! Merging as soon as these are changed.

Sure, I'll make these changes latter this evening

**Comment by Marcellofabrizio** [comment]
@arthurfiorette I did the changes you commented about. Seems we are good now.

Let me know if there is anything else!

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+labels: 
+  - bug
+assignees: arthurfiorette
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
```
[the remark this answers is not in the document]
Comment (Marcellofabrizio) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Indeed, removed

**Comment by Marcellofabrizio** [comment]
Actually thinking about it, should the Storage Type field be required @arthurfiorette ?

**Comment by arthurfiorette** [comment]
@Marcellofabrizio I think so, as you cannot use without one.

**Comment by Marcellofabrizio** [comment]
@arthurfiorette I removed the required field from the Storage type

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+    validations:
+      required: true
+  - type: input
+    id: storage
+    attributes:
+      label: What storage is being used
+      description: What storage is being used?
+      placeholder: Web Storage
+    validations:
+      required: false
```
Comment (arthurfiorette) [inline .github/ISSUE_TEMPLATE/bug_report.yml:52]:
```suggestion
      required: true
```

I'll keep it required for now...

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: Shell
```
Comment (arthurfiorette) [inline .github/ISSUE_TEMPLATE/bug_report.yml:58]:
```suggestion
      render: Shell
    validations:
      required: true
```

**Review by arthurfiorette — CHANGES_REQUESTED** [review]

**Review by arthurfiorette — APPROVED** [review]

**Comment by arthurfiorette** [comment]
Merging it now! Thanks for your effort!

**Pull request merged**